src/client.js

```diff
@@ -94,6 +94,27 @@ function Client(manager, name, config) {
 
 	if (typeof client.config.sessions !== "object") {
 		client.config.sessions = {};
+	} else {
+		// TODO: This is just for backwards compatibility
+		const newSessions = {};
+		let changed = false;
+
+		_.forOwn(client.config.sessions, (session, key) => {
+			if (key.length !== 128) {
```

Comment:
`>`? Any reason why this would be, I don't know, `null`, `undefined`, `0`, etc.?

Reply:
Non hashed keys are 96 length (48 in binary), and hashed ones are 128 (96).

Reply:
Right, and I suggested `if (key.length < 128)` (actually `>` but that was a typo lol), no big deal though, it doesn't make any difference I believe as long as hashed keys are _always_ 128 char long.